gazelle/python/parser.go

```diff
@@ -164,9 +165,32 @@ func (p *python3Parser) parse(pyFilenames *treeset.Set) (*treeset.Set, map[strin
 				mainModules[res.FileName].Add(m)
 			}
 		}
+
+		// Collect all annotations from each file into a single annotations struct.
+		for k, v := range annotations.ignore {
+			allAnnotations.ignore[k] = v
+		}
```

Comment:
`annotations.ignore` is not used in `generate.go` so we could leave this out if wanted. Thoughts? Leaving it out would mean that the returned `allAnnotations` is not representative of everything that was parsed.